src/algebra/algebra/basic.lean

```diff
@@ -1135,6 +1135,8 @@ end of_linear_equiv
 
 @[simp] lemma mul_apply (e₁ e₂ : A₁ ≃ₐ[R] A₁) (x : A₁) : (e₁ * e₂) x = e₁ (e₂ x) := rfl
 
+@[simp] lemma inv_eq_symm {a : A₁ ≃ₐ[R] A₁} : a⁻¹ = a.symm := rfl
```

Comment:
Should we instead just feed `inv` to the `simps` on L1125, treating it the same way as `mul` and `one`?

Reply:
Good point, I've done that. I think this also adresses @riccardobrasca comment?

Reply:
Yes, and I agree with Johan about `simps`.